src/store/MLTaskStore.js

```diff
@@ -86,7 +89,6 @@ const MLTaskStore = types.model('MLTaskStore', {
       const message = err && err.toString() || undefined
       self.status = ASYNC_STATES.ERROR
       self.statusMessage = message
-      self.data = data
```

Comment:
Why isn't the response data saved in the store?

Reply:
Oh, this was a mistake I was correcting. I had previously copied the `self.whatever = ...` block from the "onReceiveData" section, but this ☝️  is the _"onError"_ section. There's no `data` to be saved, and that was causing an error in the error catch. 😞